lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -104,5 +104,11 @@ public interface IFileSystem
         /// <returns></returns>
         /// <exception cref="IOException"></exception>
         IEnumerable<Uri> GetChildren(Uri directoryUri);
+
+        /// <summary>
+        /// Returns Url prefix 
+        /// </summary>
+        /// <returns></returns>
+        string UriPrefix { get; }
```

Comment:
I would leave it as it was. I like the idea to have an API like Uri CreateUriForPath(string path).